scripts/dts/extract/dma.py

```diff
@@ -0,0 +1,188 @@
+#
+# Copyright (c) 2018 Bobby Noelte
+# Copyright (c) 2018 Song Qiang <songqiang1304521@gmail.com>
+#
+# Derived from clocks.py
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
We've deprecated the dts/extract script.  We need to moved over to scripts/dts/edtlib.py and scripts/dts/gen_defines.py

Reply:
:cry: